.development.env

```diff
@@ -27,5 +27,5 @@ CDN_ASSETS_PATH=/design-system
 ADDRESS_LOOKUP_API_URL=https://whitelodge-ai-api.census-gcp.onsdigital.uk
 COOKIE_SETTINGS_URL=#
 EQ_SUBMISSION_CONFIRMATION_BACKEND=log
-SDS_API_BASE_URL=http://localhost:5003
-OIDC_TOKEN_BACKEND=local
\ No newline at end of file
+SDS_API_BASE_URL=http://sds:5003
```

Comment:
The issue with updating this is that, this file can now not be used when running runner locally. This expects runner to be running in docker.

I suggest reverting this but in docker-compose.yml set `SDS_API_BASE_URL: "http://sds:5003"` inline which will take precedence. Then, if you use this file for local runner or when running in docker, both cases work.

Reply:
Ahh makes sense, have made that change and switched the dev compose files to use the same approach as the runner url

Reply:
Doesn't look like `docker-compose-yml` was updated.

Reply:
oops - updated as per other comment